pkg/cri/config/config.go

```diff
@@ -28,6 +28,30 @@ import (
 	"github.com/containerd/containerd/v2/pkg/deprecation"
 )
 
+const (
+	// defaultImagePullProgressTimeoutDuration is the default value of imagePullProgressTimeout.
+	//
+	// NOTE:
+	//
+	// This function is ported from kubelet/dockershim's --image-pull-progress-deadline.
```

Comment:
confused about this line of comment. which function is it referring?

Reply:
Updated. Please take a look.